pandas/core/ops.py

```diff
@@ -407,8 +407,12 @@ def _validate_datetime(self, lvalues, rvalues, name):
 
             # if tz's must be equal (same or None)
             if getattr(lvalues, 'tz', None) != getattr(rvalues, 'tz', None):
-                raise ValueError("Incompatible tz's on datetime subtraction "
-                                 "ops")
+                if len(rvalues) == 1 and np.isnat(rvalues[0]):
```

Comment:
we never use ``np.isnat``, always use ``isna``.

why is this restricted to len(..) == 1? if they are all nat should be enough

Reply:
> why is this restricted to len(..) == 1?

Because the bug being addressed is specific to scalar NaT subtraction.  Non-scalar cases are handled elsewhere.

Reply:
> we never use np.isnat, always use isna.

I don't have a strong opinion.  np.isnat is more strict, which I figure is preferable since all else equal I'd like to avoid catching extra cases by accident.

Reply:
well I do. don't use it.

Reply:
You got it buddy.  Just changed and pushed.